ignite/handlers/wandb_logger.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, *args: Any, **kwargs: Any):
                 "You man install wandb with the command:\n pip install wandb\n"
             )
         if kwargs.get("init", True):
+            kwargs.pop('init')
```

Comment:
This may not always work, by default this code will give the error:
```python
KeyError: 'init'
```
as `kwargs` may not have "init" kwargs.

Let's do that in the following way:
```python
if kwargs.get("init", True):
    kwargs.pop("init", None)
    wandb.init(*args, **kwargs)
```

Reply:
agree. updated.